client/src/dojo/game/models/TutorialStep.ts

```diff
@@ -0,0 +1,84 @@
+import { TutorialGame } from "./TutorialGame";
+
+export interface TutorialStep {
+    setup(game: TutorialGame): void;
+    checkCompletion(game: TutorialGame): boolean;
+    getInstructions(): string;
+  }
+
+
+
+  export class BasicSlidingStep implements TutorialStep {
+    setup(game: TutorialGame) {
+      // Set up a simple grid with a slidable cube
+      // You'll need to implement this based on your game's structure
+    }
+  
+    checkCompletion(game: TutorialGame): boolean {
+      // Check if the player has successfully slid the cube
+      // Implement based on your game's logic
+      return false; // Placeholder
+    }
+  
+    getInstructions(): string {
+      return "Slide the highlighted cube to the left or right";
+    }
+  }
+
+
+
+  export class HammerBonusStep implements TutorialStep {
+    setup(game: TutorialGame) {
+      // Set up a simple grid with a slidable cube
```

Comment:
empty function

Reply:
The corrections you made above, were files added at the beginning of the project they don't have anything to do with the  present tutorial they have been removed